.github/workflows/release.yaml

```diff
@@ -72,37 +77,41 @@ jobs:
 
       - uses: actions/download-artifact@v2
         with:
-          name: ibazel_linux_amd64
-          path: ./npm-staging/bin/linux_amd64/ibazel
-
-      - uses: actions/download-artifact@v2
-        with:
-          name: ibazel_darwin_amd64
-          path: ./npm-staging/bin/darwin_amd64/ibazel
-
-      - uses: actions/download-artifact@v2
-        with:
-          name: ibazel_windows_amd64.exe
-          path: ./npm-staging/bin/windows_amd64/ibazel.exe
-
+          path: ./npm-staging/bin
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
 
       - name: Install Bazelisk
         run: npm --global install @bazel/bazelisk@latest
 
+      - name: Prepare ibazel binaries
+        working-directory: ./npm-staging/bin
+        run: |
+          mv darwin_amd64/ibazel_darwin_amd64 darwin_amd64/ibazel
+          mv linux_amd64/ibazel_linux_amd64 linux_amd64/ibazel
+          mv windows_amd64/ibazel_windows_amd64.exe windows_amd64/ibazel.exe
```

Comment:
I had to re-arrange the npm package as it varied slightly from previous releases and I did not want to break the structure for folks.